deps/db/src/logseq/db/frontend/malli_schema.cljs

```diff
@@ -389,6 +389,15 @@
     (remove #(#{:block/title :logseq.property/created-from-property} (first %)) block-attrs)
     page-or-block-attrs)))
 
+(def property-history-block
+  (vec
+   (concat
+    [:map]
+    [[:block/uuid :uuid]
+     [:block/created-at :int]
```

Comment:
`d/entity` on a property-history-block, returns:
``` clojure
{:block/created-at 1735878639351,
 :block/tx-id 536870923,
 :block/uuid #uuid "677767ef-6e6d-4eaf-a07c-d4cfd23ea609",
 :logseq.property.history/block #:db{:id 137},
 :logseq.property.history/property #:db{:id 66},
 :logseq.property.history/ref-value #:db{:id 68},
 :db/id 138}
```
do history-blocks need a `:block/tags=:logseq.class/history-block`?

Reply:
Adding a tag makes it easier to use tables to organize/filter the history but I don't think it'll be used too much, also the cost to have another built-in tag and more datoms because history items could be a lot.